control-theory/README.md

```diff
@@ -2,15 +2,30 @@
 
 ## Resources
 ### Classical Control
-* [Classical Control Theory](https://www.youtube.com/watch?v=oBc_BHxw78s&list=PLUMWjy5jgHK1NC52DXXrriwihVrYZKqjk) by Brian Douglas.
-* [Control Bootcamp](https://www.youtube.com/playlist?list=PLMrJAkhIeNNR20Mz-VpzgfQs5zrYi085m) by Steve Brunton.
-* [Control of Mobile Robots](https://www.coursera.org/learn/mobile-robot/home/welcome) by Dr. Magnus Egerstedt.
+* [Classical Control Theory](https://www.youtube.com/watch?v=oBc_BHxw78s&list=PLUMWjy5jgHK1NC52DXXrriwihVrYZKqjk) by Brian Douglas.                  
+  **_Why this lecture series? :_** 
+This collection of videos is intended to supplement a first-year controls class, not replace it. My goal is to take specific concepts in controls and expand on them to provide an intuitive understanding which will ultimately make you a better controls engineer.

```

Comment:
"My goal is to ...." Change to 3rd person (looks like you've copied from his course description or something.

Reply:
yeah, the old description was no so good so I decided to take his explanation.

Reply:
I'll correct it